core/import.go

```diff
@@ -45,7 +45,6 @@ func (n *OpenBazaarNode) ImportListings(r io.ReadCloser) error {
 	indexLock := new(sync.Mutex)
 	wg := new(sync.WaitGroup)
 
-	stringPool := sync.Pool{New: func() interface{} { var s string; return s }}
```

Comment:
Also not sure why this isn't needed as well

Reply:
I don't know why it's not needed, but it's not used. `stringPool` is defined and never used. `stringSlicePool` is used though.

Reply:
Okay, so these pools are being used ineffectively. It's using fairly simple/low memory types and then is reassigning over them pool'ed versions anyways. I've removed them to make the code clearer.

Reply:
This sync.Pool usage is a pattern I've never encountered. It seems to be used to just generate zero strings values? But... it's not clear why the Pool is required instead of just inlining the var decl.